pkg/features/features.go

```diff
@@ -33,6 +36,7 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	CraneAutoscaling:           {Default: true, PreRelease: featuregate.Alpha},
 	CraneAnalysis:              {Default: true, PreRelease: featuregate.Alpha},
 	CraneNodeResource:          {Default: true, PreRelease: featuregate.Alpha},
+	CraneNodeResourceTopology:  {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
should we start from CraneNodeResourceTopology=false, as it requires to cooperate with crane-scheduler

Reply:
Sure.